### PR TITLE
Implementation of simple collect sub service and CLI

### PR DESCRIPTION
**Pull request by lumjjb**

### CODE REVIEW

**Comment by lumjjb** [comment]
@mihaimaruseac @pxp928 @mlieberman85 Updated to implement a CLI in guacone and server/client impl. PTAL! Thanks!

**Review by mihaimaruseac — APPROVED** [review]

File: cmd/guacone/cmd/collectsub_client.go [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+		defer csubClient.Close()
+
+		bytes, err := io.ReadAll(cmd.InOrStdin())
+		if err != nil {
+			logger.Fatalf("error reading input from STDIN: %v", err)
+		}
+		var entries []input.CollectEntryInput
+		err = json.Unmarshal(bytes, &entries)
+		if err != nil {
+			logger.Fatalf("unmarshallign input: %v", err)
```
Comment (pxp928) [inline cmd/guacone/cmd/collectsub_client.go:91]: nit: unmarshalling

File: pkg/collectsub/client/client.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
+package client
```
Comment (pxp928) [inline pkg/collectsub/client/client.go:1]: nit: missing header

**Review by pxp928 — COMMENTED** [review]
Looks Great! Just a spelling mistake and a header.

**Review by pxp928 — APPROVED** [review]
LGTM just have to rebase to fix the static analysis

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**